core/chains/evm/client/client_test.go

```diff
@@ -24,7 +24,6 @@ import (
 
 	commonclient "github.com/smartcontractkit/chainlink/v2/common/client"
 
-	commontypes "github.com/smartcontractkit/chainlink/v2/common/chains/client"
 	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
```

Comment:
This is a self import!
Let's drop all of the `evmclient.` prefixes, and just import `common/client` plainly as `client` instead of renaming it.

Reply:
Oh it's a _test package :facepalm: 
Well we should still take advantage and use `client` for one or the other. Aliasing both is silly.

Reply:
Yeah, I'll keep the `commonclient` and drop the alias for the evm client. I'll clean up the rest as well.